vllm/compilation/multi_output_match.py

```diff
@@ -97,7 +98,7 @@ def insert_getitems(self, tuple_node: fx.Node,
                 self.graph.call_function(operator.getitem, (tuple_node, idx))
                 for idx in indices)
 
-    def insert_auto_fn(self, op: OpOverload, kwargs):
```

Comment:
cc @ProExpertProg to check if this is accurate.

Reply:
I believe this is right, but for consistency, can we use `fx.Node` instead of `Node`, like elsewhere?